### PR TITLE
generate yaml by helm template 

### PR DESCRIPTION
**Pull request by MaazMS**
Generate yaml by helm template

### CODE REVIEW

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-13-cdf9bbb4`

**Comment by stakater-user** [comment]
@tanalam2411 Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-13-cdf9bbb4`

File: charts/workshop-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -28,74 +30,30 @@ spec:
...
+          name: kube-rbac-proxy
+          ports:
+            - containerPort: 8443
+              name: https
+        - args:
+            - --metrics-addr=127.0.0.1:8080
+            - --enable-leader-election
+          command:
+            - /manager
+          image: quay.io/stakater/workshop-operator:1.2
```
Comment (rasheedamir) [inline charts/workshop-operator/templates/deployment.yaml]: we push to docker.io and not quay

File: charts/workshop-operator/templates/deployment.yaml [hunk 2]
```diff
@@ -28,74 +30,30 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
-      - args:
-        - --secure-listen-address=0.0.0.0:8443
-        - --upstream=http://127.0.0.1:8080/
-        - --logtostderr=true
-        - --v=10
-        image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.7.0
```
Comment (rasheedamir) [inline charts/workshop-operator/templates/deployment.yaml:36]: we should use redhat one here

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-13-783a271b`

File: charts/workshop-operator/templates/deployment.yaml (hunk 1 above)
Comment (MaazMS) [reply charts/workshop-operator/templates/deployment.yaml]: I have updated image like this `image: docker.io/stakater/workshop-operator` is this correct.

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-13-3238a3d8`

File: charts/workshop-operator/templates/deployment.yaml (hunk 2 above)
Comment (MaazMS) [reply charts/workshop-operator/templates/deployment.yaml:36]: ok, i have updated it.

**Pull request merged**